m0t1fs/linux_kernel/st/m0t1fs_rpc_cancel_test.sh

```diff
@@ -19,13 +19,13 @@
 #
 
 
-. `dirname $0`/common.sh
-. `dirname $0`/m0t1fs_common_inc.sh
-. `dirname $0`/m0t1fs_client_inc.sh
-. `dirname $0`/m0t1fs_server_inc.sh
-. `dirname $0`/m0t1fs_sns_common_inc.sh
+. `dirname "$0"`/common.sh
```

Comment:
Replace "`" with $()

Reply:
After updating the branch with latest code $() was already present.